HelpSource/Classes/GUI.schelp

```diff
@@ -1,202 +1,206 @@
-class:: GUI
```

Comment:
Yikes, a lot of stuff got removed here. I think you're editing from an old version of SC.